controllers/tests/config/selfnoderemediationconfig_controller_test.go

```diff
@@ -177,25 +170,6 @@ var _ = Describe("SNR Config Test", func() {
 				}, 10*time.Second, 250*time.Millisecond).Should(Succeed())
 			})
 		})
-		When("SafeTimeToAssumeNodeRebootedSeconds is modified in Configuration", func() {
```

Comment:
I think this test should be replaced and not just removed.
Instead of setting `SafeTimeToAssumeNodeRebootedSeconds` in the calculator when the configuration is changed we are setting the whole configuration which should be verified.

Reply:
I think that's covered in reboot_test.go, no?

Reply:
I don't think so.
The idea behind this test is that both the Config reconciler and the Manager reconciler share the same instance of the calculator.
So we are verifying that a change done by one is reflected in the instance encapsulated by the other (because it's the same instance).

Feel free to point me to the relevant test in case it is covered and I've missed that.

Reply:
sorry, I meant reboot/calculator_test.go
There we create 2 different configs, and ensure that GetRebootDuration() return the correct value. The configs are reconciled and set into the calculator, they are not set manually, see suite_test.go. The only thing which is manually done is setting the config to nil in the cleanup, just to be sure that the next test doesn't use old values.